Server/mods/deathmatch/logic/CPerfStat.RPCPacketUsage.cpp

```diff
@@ -219,6 +219,7 @@ ADD_ENUM1(ADD_COLPOLYGON_POINT)
 ADD_ENUM1(REMOVE_COLPOLYGON_POINT)
 ADD_ENUM1(UPDATE_COLPOLYGON_POINT)
 ADD_ENUM1(SET_DISCORD_JOIN_PARAMETERS)
+ADD_ENUM1(COLSHAPE_POLYGON_SET_HEIGHT)
```

Comment:
Perhaps rename to `SET_COLPOLYGON_HEIGHT` for consistency.
```suggestion
ADD_ENUM1(SET_COLPOLYGON_HEIGHT)
```